From 806da90479795220de2bd1f0176547134bbcaeeb Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Fri, 14 Oct 2016 13:11:29 +0800
Subject: [PATCH 1220/5242] MLK-13344-02 ARM: dts: imx: update the setpoints
 on imx6sll

commit  2c4ea4164e5d5a0d5a60cbd6008095eba06fde34 from
https://source.codeaurora.org/external/imx/linux-imx.git

According to datasheet Rev.B,06/2016 of i.MX6SLL. It has below
setpoints support:
    996MHz    1.2V
    792MHz    1.15V
    396MHz    1.05V
    198MHz    0.95V
We add a 25mV margin to cover the IR drop and board tolerance.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sll.dtsi |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sll.dtsi b/arch/arm/boot/dts/imx6sll.dtsi
index 095c9af..a543aa0 100644
--- a/arch/arm/boot/dts/imx6sll.dtsi
+++ b/arch/arm/boot/dts/imx6sll.dtsi
@@ -50,15 +50,17 @@
 			next-level-cache = <&L2>;
 			operating-points = <
 				/* kHz    uV */
-				996000  1275000
+				996000  1225000
 				792000  1175000
-				396000  975000
+				396000  1075000
+				198000	975000
 			>;
 			fsl,soc-operating-points = <
 				/* ARM kHz      SOC-PU uV */
 				996000          1225000
 				792000          1175000
 				396000          1175000
+				198000		1175000
 			>;
 			clock-latency = <61036>; /* two CLK32 periods */
 			clocks = <&clks IMX6SLL_CLK_ARM>,
-- 
1.7.9.5

